include/extractor/turn_instructions.hpp

```diff
@@ -38,6 +38,40 @@ inline TurnInstruction getTurnDirection(const double angle)
     {
         return TurnInstruction::TurnSharpRight;
     }
+    if (angle >= 67 && angle < 130)
+    {
+        return TurnInstruction::TurnRight;
+    }
+    if (angle >= 130 && angle < 158)
+    {
+        return TurnInstruction::TurnSlightRight;
+    }
+    if (angle >= 158 && angle < 202)
+    {
+        return TurnInstruction::GoStraight;
+    }
+    if (angle >= 202 && angle < 230)
+    {
+        return TurnInstruction::TurnSlightLeft;
+    }
+    if (angle >= 230 && angle < 292)
+    {
+        return TurnInstruction::TurnLeft;
+    }
+    if (angle >= 292 && angle < 336)
+    {
+        return TurnInstruction::TurnSharpLeft;
+    }
+    return TurnInstruction::UTurn;
+}
```

Comment:
Not any more. I've modified the ranges to be not of the same size any more (e.g. go straight is narrower than left/right)